### DISCUSSION TITLE
catch/mania editor - Option to not move current time to newly placed object

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#24010] [Ideas] [answered]
I'm constantly having to scroll backwards directly after I place something, as I need to look at the previous object to know whether or not the new object fits with the rest of the map.

https://github.com/ppy/osu/assets/30382015/3b1792a0-ee90-4724-ba9d-f4d26e339f6a

**Comment by peppy** [comment]
![Uploading osu! 2023-06-23 at 06.11.33.png…]()

**Reply by peppy** [reply]
![Uploading osu! 2023-06-23 at 06.11.33.png…]()

**Reply by peppy** [reply, marked as answer]
![osu! 2023-06-23 at 06 11 33](https://github.com/ppy/osu/assets/191335/85c8f737-1b26-47db-90bb-bd2df3e5ff7f)

**Reply by Xinnoh** [reply]
sorry
